pandas/tests/frame/test_indexing.py

```diff
@@ -396,8 +396,9 @@ def test_getitem_setitem_ix_negative_integers(self):
         assert (self.frame['D'] == 0).all()
 
         df = DataFrame(np.random.randn(8, 4))
-        with catch_warnings(record=True):
-            assert isna(df.ix[:, [-1]].values).all()
+        # ix does label-based indexing when having an integer index
+        with pytest.raises(KeyError):
```

Comment:
can you also test on the row dim (maybe make these 2 a separate test), may already be an existing test

Reply:
(done)